integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/table/CarbonCreateTableCommand.scala

```diff
@@ -122,7 +123,9 @@ case class CarbonCreateTableCommand(
           val tablePath = tableIdentifier.getTablePath
           val carbonRelation = CarbonSparkUtil.createCarbonRelation(tableInfo, tablePath)
           val rawSchema = CarbonSparkUtil.getRawSchema(carbonRelation)
-          sparkSession.sparkContext.setLocalProperty(EXECUTION_ID_KEY, null)
+          if (!SparkUtil.isSparkVersionXandAbove("2.3")) {
```

Comment:
Better move this to SparkUtil.scala and call from all places.

Reply:
@ravipesala Handled the comment..please review